features/help_me_choose.feature

```diff
@@ -0,0 +1,52 @@
+Feature: Help me choose
+
+  As a user, i want to use search filters, because they give a quick and easy way to find a car that suits me.
+
+  Background:
+    Given help me choose page is opened
+
+  @bodystyle
+  Scenario Outline: Browse by body style
+    When user choose <body_style> filter
+    Then user is redirected to a search page with selected <body_style> filter
+
+    Examples:
+      | body_style  |
+      | Hatchback   |
+      | Saloon      |
+      | Estate      |
+      | MPV         |
+      | SUV         |
+      | Convertible |
+      | Coupe       |
+
+  @lifestyle
+  Scenario Outline: Browse by lifestyle
+    When user choose <lifestyle> filter
+    Then user is redirected to a search page with selected <body_type> filter
+      And an Price range filter is seted as <price_range>
+      And an Seats filter is seted as <seats>
+      And an Doors filter is seted as <doors>
+      And an Boot size filter is seted as <boot_size>
+      And an Gearbox filter is seted as <gearbox>
+      And an Fuel consumption filter is seted as <fuel_consumption>
+      And an Acceleration filter is seted as <acceleration>
+      And an Insurance filter is seted as <insurance>
+      And an First year tax from filter is seted as <first_year_tax_from>
+
+    Examples:
+      | lifestyle     | price_range       | body_type | seats              | doors                 | boot_size                    | gearbox | fuel_consumption     | acceleration             | insurance               | first_year_tax_from |
```

Comment:
It's to read such big table. Is it possible to use vertical table?

Reply:
I have tried, but ...